tests/plotting/test_plotting.py

```diff
@@ -1254,117 +1272,61 @@ def test_plot_rgb():
     plotter.show(before_close_callback=verify_cache_image)
 
 
-def setup_multicomponent_data():
-    """Create a dataset with vector values on points and cells."""
-    data = pyvista.Plane()
-
-    vector_values_points = np.empty((data.n_points, 3))
-    vector_values_points[:, :] = [3.0, 4.0, 0.0]  # Vector has this value at all points
-
-    vector_values_cells = np.empty((data.n_cells, 3))
-    vector_values_cells[:, :] = [3.0, 4.0, 0.0]  # Vector has this value at all cells
-
-    data['vector_values_points'] = vector_values_points
-    data['vector_values_cells'] = vector_values_cells
-
-    return data
-
-
-def test_vector_array_with_cells_and_points():
+def test_vector_array_with_cells_and_points(multicomp_poly):
     """Test using vector valued data with and without component arg."""
-    data = setup_multicomponent_data()
-
     # test no component argument
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_points')
-    p.show()
+    # p = pyvista.Plotter()
+    # p.add_mesh(multicomp_poly, scalars='vector_values_points')
+    # p.show()
 
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_cells')
-    p.show()
+    # p = pyvista.Plotter()
+    # p.add_mesh(multicomp_poly, scalars='vector_values_cells')
+    # p.show()
 
     # test component argument
     p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_points', component=0)
+    p.add_mesh(multicomp_poly, scalars='vector_values_points', component=0)
     p.show()
 
-    p = pyvista.Plotter()
-    p.add_mesh(data, scalars='vector_values_cells', component=0)
-    p.show()
+    # p = pyvista.Plotter()
+    # p.add_mesh(multicomp_poly, scalars='vector_values_cells', component=0)
+    # p.show()
```

Comment:
Did you mean to remove or uncomment this test?

Reply:
Thanks for catching that. Added back in c619b9f7f173e1bb1bba935599258a17c2ab7557.